config/rbac/role.yaml

```diff
@@ -49,6 +49,14 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
did you update this manually? we should do it through kubebuilder annotations
Also, could we scope this down to eksa-system ns?